sample-apps/dotnet/asp_remote_service/appsettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
```

Comment:
Should we put the log level to debug or something to that effect for the future? We should only be looking at these logs if we need to debug.

Reply:
At least in E2E Test workflow, the log is not stored. So no diff I think

Reply:
We intend to do eventually dump the logs to AWS somewhere (S3, CWLogs), so we should do it anyways. Also this would be useful for debugging if an oncall is able to gather the logs during a test run if needed